backends/ebpf/ebpf_runtime.c

```diff
@@ -0,0 +1,55 @@
+#include <string.h>
+#define PCAP_DONT_INCLUDE_PCAP_BPF_H
+#include <pcap/pcap.h>
+#include "test.h"
+
+int main(int argc, char **argv) {
+    pcap_t *in_handle;
+    pcap_dumper_t *out_handle;
+    const unsigned char *packet;
+    char errbuf[PCAP_ERRBUF_SIZE];
+    struct pcap_pkthdr *pcap_hdr;
+
+    /* Skip over the program name. */
+    ++argv; --argc;
+    if ( argc != 1 ) {
+        fprintf(stderr, "Missing input trace file, exiting...\n");
+        return EXIT_FAILURE;
+    }
+
+    /* Initialize the registry of shared tables */
+    struct bpf_map_def* current = tables;
+    while (current->name != 0) {
+        registry_add(current);
```

Comment:
You should also check the result once you have one.

Reply:
done